components/style/values/computed/image.rs

```diff
@@ -454,3 +455,73 @@ impl ToComputedValue for specified::LengthOrPercentageOrKeyword {
         }
     }
 }
+
+#[derive(Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+pub enum AngleOrCorner {
```

Comment:
Well, specified `AngleOrCorner` is inside of image.rs and it looks like it's related to image. Shouldn't it be here? We can reimport it in mod.rs to use like `computed::AngleOrCorner` I think.